src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -11,7 +11,7 @@
 import seedu.address.model.person.Person;
 import seedu.address.model.person.exceptions.DuplicatePersonException;
 
-/**
```

Comment:
Why are these asterisks deleted? Not that it is a bad thing, but maybe they were there for a reason?

Reply:
Yes I realised that I had unnecessarily deleted it.